intranet/templates/eighth/emails/transferred_activity.html

```diff
@@ -0,0 +1,11 @@
+<html><body>
+    {% with description="Room changes" target=info_link name="View status" %}
+        {% include "email_metadata.html" %}
+    {% endwith %}
+
+    <p>
+        You were transferred by the 8th Period Office into {{ dest_act.activity }} on {{ date_str }} during {{ dest_act.block.block_letter }} block. Your previous signup has been removed, but you may re-signup on Ion if you wish.
```

Comment:
```suggestion
        You were transferred by the 8th Period Office into {{ dest_act.activity }} on {{ date_str }} during {{ dest_act.block.block_letter }} block. Your previous signup during this block has been removed, but you may re-signup on Ion if you wish.
```
Just a minor clarification.